### DISCUSSION TITLE
Does this library support AdMob bidding ads?

### DISCUSSION THREAD

**Discussion by manishram** [invertase/react-native-firebase#5747] [Q&A] [answered]
I am trying to integrate Mediation with Admob for Facebook ads Network.
When we are using a bidding system then we need to **Initialize the Mobile Ads SDK**. If we don't do this then it will send platform **unknown** to Facebook. 

Right now I face this issue => platform **unknown** to Facebook ads bidding. 

So there is a good chance that the reason being the same.

So I want to ask if [Initialize the Mobile Ads SDK](https://developers.google.com/admob/android/mediate?fbclid=IwAR0lPacNTfz6jHNgVJYOpje1FuLfLGWU_x8tF6Q7C78q8fyipLf8aHgOvtw#initialize_the_mobile_ads_sdk) is already part of this library or do I need to do it myself?

If I need to do it myself then where should I add the code to initialize given in the above URL?

**Comment by mikehardy** [comment, marked as answer]
Hi there! Please see this #5329 for our plans with regard to AdMob - the replacement library is still under construction but it's nearing completion
